src/main/features/ccj/draft/DraftCCJService.ts

```diff
@@ -0,0 +1,25 @@
+import * as express from 'express'
+
+import { DraftMiddleware } from 'common/draft/draftMiddleware'
+import { DraftCCJ } from 'ccj/draft/DraftCCJ'
+
+const deserialize = (value: any): DraftCCJ => {
+  return new DraftCCJ().deserialize(value)
+}
+
+const middleware = new DraftMiddleware<DraftCCJ>('ccj', deserialize)
+
+export class DraftCCJService {
+
+  static retrieve (req: express.Request, res: express.Response, next: express.NextFunction): void {
```

Comment:
The name is called retrieve, but it's a `void`. Maybe it would be worth to rename?